boltdesignsystem/bolt
# Feature/tabs adaptive menu

## Jira

http://vjira2:8080/browse/BDS-1857

## Summary

Add adaptive menu to Tabs component, based on Priority Nav.

## Details

## How to test
Check out branch and test in Chrome and IE, especially [Tabs Content](https://boltdesignsystem.com/pattern-lab/?p=components-tabs-content) page to see how component responds at smaller screen sizes.
